backend/apps/vacancies/api/v1/serializers.py

```diff
@@ -0,0 +1,35 @@
+from apps.vacancies.models import Currency, Vacancy
+from django.utils.translation import gettext_lazy as _
+from rest_framework import serializers
+from taggit.serializers import TaggitSerializer, TagListSerializerField
+
+
+class VacancySerializer(serializers.ModelSerializer, TaggitSerializer):
+    """Serializer for CRUD Cacancy endpoint."""
+
+    keywords = TagListSerializerField()
+    end_date = serializers.DateField()
+    start_date = serializers.DateField()
+    salary_max = serializers.IntegerField()
+    salary_min = serializers.IntegerField()
```

Comment:
Could you, please, explain, why you declared these fields?
Vacancy model has already had them required and you're not adding anything in here.